lib/elliptic/curves.js

```diff
@@ -7,7 +7,7 @@ var elliptic = require('../elliptic');
 
 var assert = elliptic.utils.assert;
 
-function PresetCurve(options) {
+function PresetCurve(options, skipValidation) {
```

Comment:
I feel like this should be part of the `options` object.

Reply:
agree, i separated it so it wouldn't get passed through to elliptic.curve.short/elliptic.curve.mont/elliptic.curve.edwards, but i can merge it into options if u prefer

Reply:
updated